lib/ruby_indexer/lib/ruby_indexer/index.rb

```diff
@@ -185,9 +185,11 @@ def index_all(indexable_paths: RubyIndexer.configuration.indexables, &block)
     sig { params(indexable_path: IndexablePath, source: T.nilable(String)).void }
     def index_single(indexable_path, source = nil)
       content = source || File.read(indexable_path.full_path)
+      dispatcher = Prism::Dispatcher.new
```

Comment:
While I agree with the idea, I prefer leaving this change to the PR where we're actually doing so as it will make the change linked its dependencies. If we do it in this PR, it'd be hard to understand the intention without checking the PR discussions.